web/template/vod_course_list.gohtml

```diff
@@ -96,7 +96,7 @@
             </li>
         {{end}}
     {{end}}
-    {{if and .User $course.HasRecordings}}
+    {{if and $user $course.HasRecordings}}
```

Comment:
Good catch, can you also remove the corresponding `User` field from `CoursePageData`? I don't think it's used anywhere else.